core/src/test/java/org/elasticsearch/action/IndicesRequestIT.java

```diff
@@ -223,6 +228,18 @@ public void testDelete() {
         assertSameIndices(deleteRequest, deleteShardActions);
     }
 
+    // for delete index throws an IndexNotFoundException if the index does not exist or create the index if external version is used
+    public void testDeleteAction() throws InterruptedException, ExecutionException {
```

Comment:
@bleskes thanks for the tip, I will add a `TransportBulkActionTests`.

Hm... It appeared to me that at the time the issue was opened, the `delete` was not using `BulkAction` ( based on some comments ). So I decided to add the test here to verify that explicitly using the `delete` endpoint will not create the index. But of course I can remove it ;)